jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,138 @@
+:experimental:
+:icons: font
+:page-layout: classic-docs
+:page-liquid:
+
+CircleCI pipelines are the highest-level unit of work, encompassing a project's full configuration (for example, `.circleci/config.yml`, or multiple configuration files if that is how your project is set up). Pipelines coordinate the execution of workflows and jobs to build, test, and deploy your code. Workflows can be conditional, based on xref:workflows#using-contexts-and-filtering-in-your-workflows[filters], xref:workflows#workflows-configuration-examples[job requirements], and xref:selecting-a-workflow-to-run-using-pipeline-parameters#[pipeline parameters].
+
+Pipelines include your workflows, which coordinate your jobs. Pipelines have a fixed, linear lifecycle, and are associated with a specific actor. Pipelines trigger when a change is pushed to a project that has a CircleCI configuration file included, and can also be scheduled, triggered manually through the CircleCI app, or using the API.
+
+== Dashboard
+When visiting your CircleCI dashboard, you are shown a list of recently-triggered pipelines for your organization.
+
+image::/docs/assets/img/docs/pipelines-dashboard.png[Screenshot of the pipelines dashboard in the CircleCI app]
+
+[#pipeline-architecture]
+== Pipeline architecture
+
+A pipeline is composed of workflows, which are composed of jobs. By navigating from a pipeline to a specific job, you can access your job output, test results, and artifacts through several tabs.
+
+image::/docs/assets/img/docs/pipelines-job-step-test-artifact.png[Job tab options in the CircleCI web app]
+
+The output of each job can be opened in a new tab (in either raw or formatted styling) with a unique link, making it shareable between team members.
+
+image::/docs/assets/img/docs/pipelines-job-output.png[Download and share job step output]
+
+== Configuration sources and project triggers
+
+NOTE: The information in this section on configuration sources and triggers applies to accounts integrated with the xref:github-apps-integration#[CircleCI GitHub App] or xref:gitlab-integration#[GitLab]. There are two GitHub integration types, either OAuth or through the GitHub App. To find out which GitHub integration type you have refer to the xref:github-apps-integration#[GitHub App integration] page.
+
+#Add some information about why people want to go and add these things. On the Multiple config files setup page for GitHub app we have: You can use multiple configuration files in a CircleCI project and set up separate triggers for each one. Each configuration triggers a separate pipeline on a trigger event. If your existing configuration file is getting too large to manage, consider splitting it into multiple files.#
+
+=== Configuration source
+
+Configuration sources show CircleCI where a pipeline configuration for your project is stored. A configuration source can be set up for use when _triggering_ a pipeline.
```

Comment:
Configuration sources show CircleCI where a **pipeline configuration file** is stored. A configuration source can be set up for use when _triggering_ a pipeline.

Reply:
I wonder if we can go even further. Something like:
"Configuration sources define the units of work that can be run within your projects. Each configuration source defines the location of a pipeline configuration file, and can be set up to be used when _triggering_ a pipeline.